### PR TITLE
Create Bitcoin Disclaimer Page

### PR DESCRIPTION
**Pull request by bioy-casares**
@jalehman @ixv Please review.

### CODE REVIEW

**Comment by bioy-casares** [comment]
Appears to have some issues. Am unable to parse how to fix from the Netlify logs.  This noob would appreciate help from an  experienced hand. @g-a-v-i-n

File: content/bitcoin.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
++++
+title = “Bitcoin Wallet Disclaimers
```
Comment (matildepark) [inline content/bitcoin.md]:
```suggestion
title = "Bitcoin Wallet Disclaimers"
```

**Comment by jalehman** [comment]
Rename to bitcoin-wallet.md to match what I told @ixv.

—

~wolref-podlex
https://urbit.org ( https://urbit.org/ )

**Comment by bioy-casares** [comment]
> Rename to bitcoin-wallet.md to match what I told @ixv. — ~wolref-podlex https://urbit.org ( https://urbit.org/ )
> […](#)

done @jalehman

File: content/Bitcoin-wallet.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
++++
+title = "Bitcoin Wallet Disclaimers"
++++
+
+
+### Bitcoin Wallet Disclaimers
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
```
This is unnecessary due to the title above.

File: content/Bitcoin-wallet.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+### Verify
+
+You should verify your version of this Bitcoin wallet. To do so, follow the instructions in the README in our [Github](https://github.com/urbit/urbit/blob/master/pkg/bitcoin-wallet/README.md) repo. 
+
+Instructions reproduced here: 
+
+"To verify your version of the bitcoin wallet, run the following command in the dojo:
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
To verify your version of the bitcoin wallet, run the following command in the dojo:
```

File: content/Bitcoin-wallet.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+`> +btc-wallet-check`
+It should return with the following hash:
+`0v9t022.n8kv1.5emkt.s2p9i.hvsa9`
+
+
+### Statement
+
+The Urbit Bitcoin Wallet and its related components are experimental pieces of software. You are free to review their code before using: [Wallet](https://github.com/urbit/urbit/blob/master/pkg/bitcoin-wallet/README.md) [Node Provider Agents](https://github.com/timlucmiptev/btc-agents)
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
The Urbit Bitcoin Wallet and its related components are experimental pieces of software. You are free to review the open-source code before using: 
- [Wallet](https://github.com/urbit/urbit/blob/master/pkg/bitcoin-wallet/README.md) 
- [Full Node Provider Agents](https://github.com/timlucmiptev/btc-agents)
```

File: content/Bitcoin-wallet.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+No guarantees are made about the security of this software. It may contain vulnerabilities that could lead to loss of funds. This wallet software has not had a formal security security audit. 
+
+You can read about the security audit for Urbit's networking protocol in this [post](https://urbit.org/blog/security-audit/) This audit does not encompass the wallet, except to the extent that the wallet operates on Urbit's networking architecture.
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
You can read about the security audit for Urbit's networking protocol in this [post](https://urbit.org/blog/security-audit/). This audit does not encompass the wallet, except to the extent that the wallet operates on Urbit's networking architecture.
```

File: content/Bitcoin-wallet.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+If you decide to use this wallet, please store responsible amounts of Bitcoin, and use best security practices. The developers of this software cannot help you recover your keys / funds, if lost. 
+
+
+### License
+
+With the exception of its external dependencies, this software is a component of Urbit, and is released under Urbit's MIT License, reproduced below.Â  [View Original](https://raw.githubusercontent.com/urbit/urbit/master/LICENSE.txt)
```
Comment (jalehman) [inline content/Bitcoin-wallet.md]:
```suggestion
With the exception of its external dependencies, this software is a component of Urbit, and is released under Urbit's MIT License, reproduced below.  [View Original](https://raw.githubusercontent.com/urbit/urbit/master/LICENSE.txt)
```

**Review by jalehman — CHANGES_REQUESTED** [review]
Some minor formatting tweaks to apply. Otherwise, looks good to me.

**Pull request merged**